client/src/Recommend.jsx

```diff
@@ -103,13 +151,64 @@ const Recommend = ({ recommendList }) => {
           <span>Save to list</span>
           <svg stroke="currentColor" fill="none" strokeWidth="2" viewBox="0 0 24 24" strokeLinecap="round" strokeLinejoin="round" height="20px" width="20px" xmlns="http://www.w3.org/2000/svg"><polyline points="6 9 12 15 18 9" /></svg>
         </SaveButton>
-        <Button>
+        <Button ref={buttonRef} onClick={() => setShowPopper(!showPopper)}>
```

Comment:
Jonathan gave me feedback that using show in a variable name suggests that it is a function instead of a boolean and suggested using popperVisible instead for example (for me it was showCalendar --> calendarVisible)

Reply:
@claire-melbourne This is great feedback, going to change this first before I commit.